spec/api.yaml

```diff
@@ -85,6 +85,9 @@ paths:
         - onboarding
       operationId: LoginUser
       summary: 'Login User'
+      parameters:
+         - name: deviceId
```

Comment:
Let the name be x-device-id, and define it in the components.parameters instead, so it can be re-used.

Reply:
> Good work, a number of changes to be made and you should also test the docs by running the server and then go to /docs in your browser.

I have made the changes now but I am having an issue running it on my computer  " example-app@1.0.0 dev:start", This is the error it is showing.